docs/getting-started/supported-languages-and-tools.md

```diff
@@ -82,11 +83,21 @@ The table below lists all languages and frameworks that Codacy supports and the
       <td>-</td>
       <td>-</td>
     </tr>
+    <tr>
+      <td>Bash</td>
```

Comment:
@nicklem  despite Semgrep calling the language Bash, we're mapping it to Codacy's "Shell"